docs/design/extending-api.md

```diff
@@ -113,18 +113,15 @@ For example, if a user creates:
 
 ```yaml
 metadata:
-  name: cron-tab.example.com
+  name: cron-tab.stable.example.com
 apiVersion: experimental/v1alpha1
 kind: ThirdPartyResource
 description: "A specification of a Pod to run on a cron style schedule"
-versions:
-  - name: stable/v1
-  - name: experimental/v2
+version: v1
```

Comment:
I intended you to make another one of these entries for each version. I see I need to add the version to the name.